backend/app/models.py

```diff
@@ -20,3 +21,20 @@ def to_dict(self):
     def __repr__(self):
         """Method to return a string representation of the model"""
         return f"Item('{self.name}', '{self.description}')"
+
+class User(db.Model):
```

Comment:
Both User and KnowledgeBase models into separate files. I would recommend you to create a new folder called models and inside it create User model inside models/user.py and KnowledgeBase model inside models/knowledgebase.py. I also recommend you to rename KnowledgeBase  into Content and define its model inside models/content.py

- [ ] 

- [ ]